### ISSUE TITLE
bulkio: Flake in TestFullClusterBackup  (release-20.1)

### ISSUE THREAD

**Issue by knz** [cockroachdb/cockroach#52561]
Found in https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_Test/2165876

```
------- Stdout: -------
=== RUN   TestFullClusterBackup/ensure_system_table_data_restored
--- FAIL: TestFullClusterBackup/ensure_system_table_data_restored (0.03s)
full_cluster_backup_restore_test.go:180: query 'SELECT "tableID", name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
53, my_stats, {1}, 10
53, my_stats, {2}, 10
53, my_stats, {3}, 10
got:
```

cc @dt for triage

**Comment by pbardea** [comment]
I'm also seeing this flake in 19.2 - #52473. Planning on taking a look today.

**Comment by pbardea** [comment]
Closed by #53035
